### ISSUE TITLE
CLI help is not exhaustive

### ISSUE THREAD

**Issue by vviers** [signaux-faibles/predictsignauxfaibles#46]
## Problem
```sh
python -m predictsignauxfaibles --help
```

only returns 

```
usage: main.py [-h] [--model_name MODEL_NAME]

Run model prediction

optional arguments:
  -h, --help            show this help message and exit
  --model_name MODEL_NAME
                        The model to use for prediction. If not provided,
                        models/default will be used
```

All other optional arguments are not listed

## Reason

`parser.parse_args()` is called twice because default values for optional arguments depend on the value provided for the first parameter 🤯 

## Solution

default values for params must not depend on other params

**Comment by slebastard** [comment]
That was an issue with the way that we chose to use argparse with custom train/test/predict set parameters, calling `argparse.ArgumentParser.parse_args()` twice so as to process the `model_name` from the first call to get the default values of the other parameters for the second call.
After discussing possible ways forward, we will have the following configuration:
- argparse option `model_name` will have default value `default`, pointing to the default model configuration
- other options will be without a default value: `train_spl_size`, `train_from`, etc
- default values will be assigned a posteriori when these options are set to None, based on the default values imported from model_name/model_config.py

**Comment by vviers** [comment]
Fixed by #51
